cmd/gaia/app/app.go

```diff
@@ -60,7 +60,7 @@ type GaiaApp struct {
 func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, baseAppOptions ...func(*bam.BaseApp)) *GaiaApp {
 	cdc := MakeCodec()
 
-	bApp := bam.NewBaseApp(appName, cdc, logger, db, baseAppOptions...)
+	bApp := bam.NewBaseAppNoCodec(appName, logger, db, auth.DefaultTxDecoder(cdc), baseAppOptions...)
```

Comment:
We may want to revert this back to using deprecated NewBaseApp. Otherwise QueryResult will be encoded with json rather than amino. May be a breaking change even though tests pass.